packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -407,33 +462,41 @@ export class SharedConditionalsTest extends AbstractConditionalsTest {
 
     this.assertText('T1F2');
   }
+}
 
-  ['@test it maintains DOM stability when condition changes from a truthy to a different truthy value']() {
-    this.renderValues(true);
-
-    this.assertText('T1');
+export class SharedSyntaxConditionalsTest extends SharedConditionalsTest {
+  ['@htmlbars it does not update when the unbound helper is used']() {
```

Comment:
@rwjblue just curious, how difficult would it be to write a babel plugin that lets us write something like this?

```javascript
  // @test
  ["test name"] {
    ...
  }

  // @test @skip("glimmer", "unbound is not yet implemented")
  ["another test"] {
    ...
  }
```

(But maybe at that point we are really just implementing method decorators...)